Documentation/api-reference.md

```diff
@@ -208,7 +211,7 @@ When this API is called with a valid advertising identifier, the Identity for Ed
 The GAID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall.
 
 > **Warning**  
-> In order to enable collection of the user's current advertising tracking authorization selection for the provided advertising identifier, you need to install and register the [AEPEdgeConsent](https://aep-sdks.gitbook.io/docs/foundation-extensions/consent-for-edge-network) extension and update the [AEPEdge](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension) dependency to minimum 1.3.2.
+> In order to enable collection of the user's current advertising tracking authorization selection for the provided advertising identifier, you need to install and register the [AEPEdgeConsent](https://github.com/adobe/aepsdk-edgeconsent-android) extension and update the [AEPEdge](https://github.com/adobe/aepsdk-edge-android) dependency to minimum 1.3.2.
```

Comment:
AEPEdgeConsent -> Consent 
AEPEdge -> Edge
reading:
register the Consent extension and update the Edge ..